app/components/homepage-components/WorkExperienceCard.tsx

```diff
@@ -25,23 +29,43 @@ const WorkExperienceCard = ({ job }: WorkExperienceCardProps) => {
     }
   }, [theme, isMouseOver, job]);
 
+  const handleMouseLeave = () => {
+    setAnimateRight(false);
+    setTimeout(() => {
+      setIsMouseOver(false);
+    }, 200);
+  };
+
+  const handleMouseEnter = () => {
+    setAnimateRight(true);
+    setIsMouseOver(true);
+  };
+
   return (
-    <div
+    <motion.div
+      initial={{ x: "25%", opacity: 0 }}
+      whileInView={{ x: animateRight ? "1.5rem" : "0rem", opacity: 1 }}
+      transition={{
+        x: isMouseOver ? { duration: 0.2, delay: 0 } : { duration: 0.7, delay },
+        opacity: { duration: 0.7, delay },
+      }}
+      viewport={{ once: true }}
       className={`flex flex-col rounded-xl border border-white800 bg-white900 p-9 dark:border-0 dark:bg-black200 xl:flex-row ${
-        isMouseOver && "shadow-lg dark:bg-black300"
+        isMouseOver &&
+        "shadow-[-10px_10px_15px_rgba(0,0,0,0.1)] dark:bg-black300"
```

Comment:
Hard-coded color along with a pre-defined color you created... hmm... :)

Reply:
I've just changed this to a tailwind class shadow-lg

Reply:
I've just changed this to a tailwind class shadow-lg